Gemfile

```diff
@@ -1,15 +1,13 @@
 source "http://rubygems.org"
-# Add dependencies required to use your gem here.
-gem 'crack'
-gem 'aws-sdk'
 
-# Add dependencies to develop your gem here.
-# Include everything needed to run rake, tests, features, etc.
+gem 'crack', '~> 0.4.3'
+gem 'aws-sdk', '~> 2.2', '>= 2.2.24'
+
 group :development do
-  gem "rspec", ">= 0"
-  gem "bundler", "~> 1.0"
-  gem "jeweler", "~> 2.0.1"
-  gem 'guard'
+  gem 'rspec', '>= 0'
+  gem 'bundler', '~> 1.0'
+  gem 'jeweler', '~> 2.1', '>= 2.1.2'
```

Comment:
Probably not something we want to tackle now, but we're using `juwelier` in most (all?) other gems so maybe we should migrate this one as well. For reference `jeweler` gem had got unmaintained for a while so `juwelier`, a fork of that gem, was created. It seems nowadays both are supported by the same person though from [README](https://github.com/technicalpickles/jeweler/blob/master/README.markdown), the fork is more up to date:

_Jeweler maintenance has now shifted to Fred Mitchell. I am now maintaining both Jeweler and its fork, juwelier. I will keep Jeweler at least functional with the latest Ruby releases, but put new features in Juwelier. Your input on this is more than welcome._

Reply:
Aha, interesting! I was wondering about `jeweler` vs. `juwelier`. Yeah, sounds like we should migrate to `juwelier`.